### PR TITLE
Fixes #20 - Index writer SG config

### LINKED ISSUE #20: Ability to specify index writers (mix of index-writers and non-writers)
In order to be able to gather performance data as in https://github.com/couchbase/sync_gateway/issues/1272, the perfcluster work needs to be able to support "mixed mode" where:

* Certain instances are tagged as index writers, and other instances are non-writers.  
* The gateload/gatling load generators should only be pointed at the non-writers 
* The index writers will need to have their JSON config changed to set indexwriter = true (the rest of the config would be identical)

Steps that need to happen:

1. Modify sg configs and restart sgs
1. Generate gateload configs to point only at non writers

### PR DESCRIPTION
**Pull request by tleyden**
https://github.com/couchbaselabs/sync-gateway-testcluster/issues/20

### CODE REVIEW

File: provision/ansible/playbooks/tasks/start-sync-gateway.yml [hunk 1]
```diff
@@ -1,6 +1,3 @@
-# Add sync_gateway user, copy new config
-- name: add sync gateway user
-  user: name=sync_gateway createhome=yes
 - name: make service install script executable
```
Comment (sethrosetter) [inline provision/ansible/playbooks/tasks/start-sync-gateway.yml]: Could we also pull these steps out into the install-source and install-package script as well?

File: provision/ansible/playbooks/tasks/start-sync-gateway.yml [hunk 2]
```diff
@@ -8,9 +5,6 @@
 - name: change permissions of sync gateway executable
   file: path=/home/sync_gateway/sync_gateway mode=a+x
 - debug: msg="Starting sync_gateway with config {{ sync_gateway_config_filepath }}"
```
Comment (sethrosetter) [inline provision/ansible/playbooks/tasks/start-sync-gateway.yml]: Could we also pull these steps out into the install-source and install-package script as well?

**Comment by sethrosetter** [comment]
@tleyden please rebase on master. Thanks!

**Comment by tleyden** [comment]
@sethrosetter ok this is rebased!  Can you implement the change in your suggestion and push up a new commit?

**Pull request merged**